prover_rust/src/types.rs

```diff
@@ -129,8 +101,9 @@ pub struct ProofDetail {
     pub id: String,
     #[serde(rename = "type", default)]
     pub proof_type: ProofType,
-    pub chunk_proof: Option<ChunkProof>,
-    pub batch_proof: Option<BatchProof>,
+    // pub chunk_proof: Option<String>,
+    // pub batch_proof: Option<String>,
```

Comment:
```suggestion
```

Reply:
will change this in refactor/prover branch